Extensions/xkit_preferences.js

```diff
@@ -1080,10 +1103,9 @@ XKit.extensions.xkit_preferences = new Object({
 			obj.icon = XKit.extensions.xkit_preferences.default_extension_icon;
 		}
 
-		if (XKit.installed.check(obj.name) === true) { return ""; }
-		if (obj.name.startsWith("xkit_")) { return ""; }
-
-		var m_html = '<div class="xkit-gallery-extension" id="xkit-gallery-extension-' + obj.name + '" data-extension-id="' + obj.name + '">' +
+		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
+		
+		var m_html = '<div class="xkit-gallery-extension' + (XKit.installed.check(obj.name) ? " xkit-installed-extension" : "") + '" id="xkit-gallery-extension-' + obj.name + '" data-extension-id="' + obj.name + '">' +
```

Comment:
Not a huge fan of inlining the class names like this.  Maybe split the class name generation out into a variable and then just inline the variable.